content/updates/2020-08/2-API-search.md

```diff
@@ -0,0 +1,36 @@
+---
+pim_announcement_img: ./img/api-filters.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Only get what you need thanks to our new API filter
```

Comment:
To be straight to the point, I would suggest:
```suggestion
# Efficient catalog structure export via API thanks to new filters
```

Reply:
```suggestion
# New filters to simplify API catalog structure exports
```